Makefile

```diff
@@ -3,60 +3,56 @@
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 mkfile_dir := $(dir $(mkfile_path))
 TESTS_DIR ?= $(mkfile_dir)tests
-# directory with all the "template" files used to generated the files used during
-# the tests.
+# directory with all the "template" files used to generated the files used during the tests
 ROOT_RESOURCES_DIR ?= $(mkfile_dir)resources
-# timeout for the kubectl commands
-TIMEOUT ?= 5m
```

Comment:
Don't the timeout configuration removal causes issues when running in the slower Github runners?

Reply:
it's not used in tests, I see no difference when running without it